Streetcode/Streetcode.XUnitTest/MediatRTests/StreetCode/Terms/CreateTermHandlerTests.cs

```diff
@@ -27,7 +27,7 @@ public CreateTermHandlerTests()
 			_mockMapper = new();
 			_mockRepository = new();
             _mockLogger = new Mock<ILoggerService>();
-            _mockLocalizerCannotConvertNull = new Mock<IStringLocalizer<CannotConvertNullSharedResource>>();
+			_mockLocalizerCannotConvertNull = new Mock<IStringLocalizer<CannotConvertNullSharedResource>>();
```

Comment:
Why the long space?)

Reply:
![image](https://github.com/ita-social-projects/StreetCode/assets/61435019/f5b6736e-cbc8-48b2-840f-e7081dfc51d5)
Everything correct